Signal/src/call/CallService.swift

```diff
@@ -77,8 +77,8 @@ enum CallError: Error {
     case timeout(description: String)
 }
 
-// FIXME TODO do we need to timeout?
-fileprivate let timeoutSeconds = 60
+// Should be roughly synced with Android client for consistency
+fileprivate let timeoutSeconds = 120
```

Comment:
Maybe rename to `connectingTimeoutSeconds`?  This doesn't apply once you're connected, right?